static/templates/confirm_user_deactivation_modal.hbs

```diff
@@ -0,0 +1,14 @@
+<div id="confirm_user_deactivation_modal" class="modal modal-bg hide fade" tabindex="-1" role="dialog" aria-labelledby="confirm_user_deactivation_modal_label" aria-hidden="true">
+    <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
+        <h3 id="confirm_user_deactivation_modal_label">{{t "Deactivate" }} <span class="email"></span></h3>
+    </div>
+    <div class="modal-body">
+        <p>{{#tr this}}By deactivating <strong><span class="user_name"></span></strong> &lt;<span class="email"></span>&gt;, they will be logged out immediately.{{/tr}}</p>
+        <p>{{t "Their password will be cleared from our systems, and any bots they maintain will be disabled." }}</p>
+    </div>
+    <div class="modal-footer">
+        <button class="button rounded" data-dismiss="modal">{{t "Cancel" }}</button>
+        <button class="button rounded btn-danger confirm_user_deactivation_button">{{t "Deactivate now" }}</button>
+    </div>
+</div>
```

Comment:
This looks like an almost copy of `static/templates/settings/deactivation_user_modal.hbs`; generally copying code in a large project is a bad idea as they'll inevitably diverge in buggy ways :).

@sahil839 may be able to help if you're not sure how to deduplicate this.